packages/mobile/src/components/chat-actions-drawer/ChatActionsDrawer.tsx

```diff
@@ -0,0 +1,82 @@
+import { View } from 'react-native'
+import { useDispatch, useSelector } from 'react-redux'
+
+import { Text } from 'app/components/core'
+import { NativeDrawer } from 'app/components/drawer'
+import { useNavigation } from 'app/hooks/useNavigation'
+import type { AppState } from 'app/store'
+import { getData } from 'app/store/drawers/selectors'
+import { setVisibility } from 'app/store/drawers/slice'
+import { makeStyles } from 'app/styles'
+
+const CHAT_ACTIONS_MODAL_NAME = 'ChatActions'
```

Comment:
no action necessary on this as I know it's already an established pattern, but it does concern me slightly that we have the overloading of the word `Action` for `ActionDrawer` 😬